src/server/routers/game.ts

```diff
@@ -107,7 +109,7 @@ export const gameRouter = router({
     )
     .mutation(async ({ input }) => {
       const game = await prisma.game.create({
-        data: { ...input, default_settings: {} },
+        data: { ...input },
```

Comment:
Minor performance improvement to not spread the input
```suggestion
        data: input,
```